webapp/src/components/call_widget/widget_banner.tsx

```diff
@@ -165,8 +182,25 @@ const Footer = styled.div`
 `;
 
 const ConfirmButton = styled.button`
-  color: var(--button-bg);
-  font-weight: 600;
+  &&& {
```

Comment:
Do we actually need the extra specificity here?

Reply:
At the moment we do unless we refactor some stuff as it was conflicting with definitions in https://github.com/mattermost/mattermost-plugin-calls/blob/da019884e890fed6a2333287c176f3504c16c706/webapp/src/components/call_widget/component.scss